time stamp

string uav_name
geometry_msgs/Point position
float64 heading
bool flying_normally

int32   var_int1
int32   var_int2
float64 var_dbl
